hunting-buddy.lic

```diff
@@ -54,31 +54,36 @@ class HuntingBuddy
       duration = info[:duration]
       stop_on_skills = info['stop_on']
 
-      # Execute scripts to run before the hunt
-      before_actions.each do |before_action|
-        action_parts = before_action.split(' ')
-        script_name = action_parts.shift
-        wait_for_script_to_complete(script_name, action_parts)
-      end
+      #Execute scripts to run before the hunt
+      execute_actions(before_actions)
 
-      next if all_skills_at_cap?(stop_on_skills)
+      if all_skills_at_cap?(stop_on_skills)
```

Comment:
Quick poll on lnet says that if someone is entering a hunt when all skills are capped then they have an incredibly inefficient setup and wouldn't be too inconvenienced by it being made slightly more inefficient with needles pre-buffing